### PR TITLE
OP-1562: fix issue with refresh

### PR DESCRIPTION
**Pull request by olewandowski1**
[OP-1562](https://openimis.atlassian.net/browse/OP-1562)

Changes:
- Fix reload functionality. It was crashing the form: 
If editing an existing item, it fetches the item's details.
If edited item was saved, it fetches recent version of item.
If creating a new item and not saved, resets the form.
If a new item was created and saved, fetches and navigates to the item's details.

[OP-1562]: https://openimis.atlassian.net/browse/OP-1562?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -231,18 +231,18 @@ export function newMedicalItem() {
 
 export function fetchMedicalServiceMutation(mm, clientMutationId) {
   const payload = formatPageQuery(
-    "mutationLogs",
+    "medicalServices",
     [`clientMutationId:"${clientMutationId}"`],
-    ["id", "medicalServices{id}"],
+    ["uuid"],
```
Comment (wzglinieckisoldevelo) [inline src/actions.js:236]: Are you sure this is not used by other modules like claim etc? It seems to me that something could brake in other modules.

Comment (olewandowski1) [reply src/actions.js:236]: This is used only here. Moreover, this was never working properly. There is no argument like _medicalServices_ on _mutationLogs_ query.

**Comment by wzglinieckisoldevelo** [comment]
LGTM

**Review by jdolkowski — APPROVED** [review]
New code regarding fetching an instance using mutationId is compliant with the backend logic to do so.

**Pull request merged**